### PR TITLE
Add set-nth to the list functions group in the function documentation

### PR DESCRIPTION
**Pull request by xzyfer**
This PR updates the functions reference documentation by adding `set-nth` to the list functions table.

screen shot 2014-10-29 at 3 28 02 pm

### CODE REVIEW

File: lib/sass/script/functions.rb [hunk 1]
```diff
@@ -170,6 +170,9 @@ module Sass::Script
   # \{#nth nth($list, $n)}
   # : Returns a specific item in a list.
   #
+  # \{#length set-nth($list, $n, $value)}
```
Comment (xzyfer) [inline lib/sass/script/functions.rb:173]: Sorry @nex3 I just noticed I failed at copy paste. This should be `# \{#set-nth set-nth($list, $n, $value)}`

Comment (nex3) [inline lib/sass/script/functions.rb:173]: Can you send another PR to fix this?

Comment (xzyfer) [inline lib/sass/script/functions.rb:173]: Yep, done. https://github.com/sass/sass/pull/1501
